src/app/dynamic/enterprise/metering/old-reports/list/template.html

```diff
@@ -0,0 +1,100 @@
+<!--
+               Kubermatic Enterprise Read-Only License
+                      Version 1.0 ("KERO-1.0”)
+                  Copyright © 2020 Kubermatic GmbH
+
+1. You may only view, read and display for studying purposes the source
+   code of the software licensed under this license, and, to the extent
+   explicitly provided under this license, the binary code.
+2. Any use of the software which exceeds the foregoing right, including,
+   without limitation, its execution, compilation, copying, modification
+   and distribution, is expressly prohibited.
+3. THE SOFTWARE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND,
+   EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+   MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+   IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY
+   CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+   TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+   SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+END OF TERMS AND CONDITIONS
+-->
+
+<table class="km-table"
+       mat-table
+       matSort
+       matSortActive="name"
+       matSortDirection="asc"
+       [dataSource]="dataSource">
+  <ng-container [matColumnDef]="column.name">
+    <th mat-header-cell
+        *matHeaderCellDef
+        class="km-header-cell p-60"
+        mat-sort-header>Name</th>
+    <td mat-cell
+        *matCellDef="let element">{{element.name}}</td>
+  </ng-container>
+
+  <ng-container [matColumnDef]="column.size">
+    <th mat-header-cell
+        *matHeaderCellDef
+        class="km-header-cell p-20"
+        mat-sort-header>Size</th>
+    <td mat-cell
+        *matCellDef="let element">{{element.size | kmSize}}</td>
+  </ng-container>
+
+  <ng-container [matColumnDef]="column.lastModified">
+    <th mat-header-cell
+        *matHeaderCellDef
+        class="km-header-cell p-20"
+        mat-sort-header>Created</th>
```

Comment:
Should we label this column `Updated`/`Last Updated` since we're displaying the last modified time?

Reply:
Hmm, technically if someone touches a file on S3 this value may change, but in 99% of cases this `Last modified` is same as `Created`. It's just that S3 doesn't have created value, whenever you create sth new it sets updated (Last Modified) value with creation timestamp. I think @cschieder has decided in the past that displaying this is more user friendly.

Reply:
I'd be fine with using "Last modified" too, however there's the argument that if it's 99% of the time the creation time then "Created" could be easier to understand, especially in the context of reports.